packages/sdks/snippets/remix/app/routes/($slug)._index.tsx

```diff
@@ -0,0 +1,46 @@
+// ($slug)._index.tsx
+import {
+  Content,
+  fetchOneEntry,
+  getBuilderSearchParams,
+  isPreviewing,
+  type BuilderContent,
+} from '@builder.io/sdk-react';
+import type { LoaderFunctionArgs } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+import { fetch as webFetch } from '@remix-run/web-fetch';
+
+const apiKey = 'ee9f13b4981e489a9a1209887695ef2b';
+
+export const loader = async ({ params, request }: LoaderFunctionArgs) => {
+  const url = new URL(request.url);
+  const urlPath = `/${params['slug'] || ''}`;
+
+  const page = await fetchOneEntry({
+    model: 'page',
```

Comment:
nit: can you add a model name variable similar to how you've handled in all of the other pages and re-use it?